crane-cloud/backend
# Check and validate flutter-wave payment status

# Description

- Validate the payment status whether successful or not.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

- https://trello.com/c/vTy3FIxq

## How Can This Be Tested?

- Fetch the branch and test the get request using the transaction_id of the transaction whose status you want to verify in the query params.


## Checklist:
